java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3564,62 +3573,6 @@ public void xgroupCreateConsumer_xreadgroup_xgroupDelConsumer(BaseClient client)
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
-    @SneakyThrows
-    @ParameterizedTest(autoCloseArguments = false)
-    @MethodSource("getClients")
```

Comment:
was it a duplicating test?

Reply:
ignore this... it's a rebasing artefact/merge conflict.  
But yeah, somehow the test got duplicated.  The above test is the one that needs to stick around.